mavros/src/plugins/local_position.cpp

```diff
@@ -46,7 +47,8 @@ class LocalPositionPlugin : public MavRosPlugin {
 		lp_nh.param<std::string>("tf/frame_id", tf_frame_id, "local_origin");
 		lp_nh.param<std::string>("tf/child_frame_id", tf_child_frame_id, "fcu");
 
-		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local", 10);
+		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local/position", 10);
+		local_velocity = lp_nh.advertise<geometry_msgs::TwistStamped>("local/velocity", 10);
```

Comment:
I just do not see reason for local_position/**local**/velocity. Obviously it is already *local*.
Maybe it may have sense to rename `local_position/local` to `local_position/pose`. But really you see topic name and it's type so i suppose keeping current name require less effort.